Source/WarpX.H

```diff
@@ -167,6 +167,7 @@ public:
     static bool use_filter;
     static bool use_kspace_filter;
     static bool use_filter_compensation;
+    static bool use_damp_in_z_guard;
```

Comment:
I could change it to `use_damp_fields_in_z_guard`. I don't think that would be too long - I prefer long and precise rather than short and unclear. I mostly copy and paste (instead of typing) such parameters so it doesn't matter so much how long they are.